backend/README.md

```
@@ -0,0 +1,169 @@
+   npm run dev:watch
+
+   # Or production build
+   npm run start
+   ```
+
+The API will be available at `http://localhost:3000`
+
+## 📚 API Documentation
+
+Once the application is running, visit:
+- **Swagger UI:** http://localhost:3000/api/docs
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)